Changes

```diff
@@ -2,6 +2,7 @@ Revision history for WebService-MyAffiliates
 
 {{$NEXT}}
 
+0.10      2024-12-11 15:12:54+00:00 UTC
```

Comment:
Not `0.11` ?

Reply:
0.10. I released yesterday. the next version will 0.11. that's why you can see 0.11 in main module: it is the next version.